collector/src/bin/rustc-perf-collector/sysroot.rs

```diff
@@ -42,10 +41,11 @@ impl Sysroot {
         download.get_and_extract(ModuleVariant::Rustc)?;
         download.get_and_extract(ModuleVariant::Std)?;
         download.get_and_extract(ModuleVariant::Cargo)?;
+        download.get_and_extract(ModuleVariant::RustSrc)?;
 
         download.into_sysroot()
+        }
     }
-}
```

Comment:
Argh, VSCode's git integration misled me (I have some NixOS hacks in here I didn't want to commit :P).